std/file.d

```diff
@@ -2287,21 +2287,23 @@ if (isConvertibleToString!R)
 // Same as mkdir but ignores "already exists" errors.
 // Returns: "true" if the directory was created,
 //   "false" if it already existed.
-private bool ensureDirExists(in char[] pathname)
+private bool ensureDirExists(R)(R pathname)
+if (isInputRange!R && !isInfinite!R && isSomeChar!(ElementEncodingType!R) &&
+    !isConvertibleToString!R)
```

Comment:
I don't think you want `!isConvertibleToString!R` as unlike `mkdir` there is no overload with `if(isConvertibleToString!R)`.